management/connection.go

```diff
@@ -157,6 +157,101 @@ type Connection struct {
 	ShowAsButton *bool `json:"show_as_button,omitempty"`
 }
 
+// ScimConfiguration represents the SCIM configuration for a connection.
+// This struct is used primarily for enterprise connections.
+type ScimConfiguration struct {
```

Comment:
```suggestion
type SCIMConfiguration struct {
```

SCIM is an acronym for System for Cross-domain Identity Management, so it would be better to have this in CAPS.

Reply:
In Java land they leave those in camel case, but it does seem more common in Go not to